.env.example

```diff
@@ -1,2 +1 @@
-[secrets]
 TOKEN=<replace-me-please>
```

Comment:
Shouldn't `TOKEN` now be `DISCORD_TOKEN`?

Reply:
```suggestion
DISCORD_TOKEN=<replace-me-please>
```
Yes, indeed